app/src/home.js

```diff
@@ -577,27 +601,81 @@ export default function Home() {
   );
 
   let stage;
-
   if (!transcripts) {
     stage = STAGE_HOME;
-  } else if (audioStream && !showAnalysis) {
+  } else if (audioStream) {
     stage = STAGE_TRANSCRIBING;
-  } else if (!showAnalysis) {
+  } else if (!showSOAPReview) {
     stage = STAGE_TRANSCRIBED;
   } else if (!showExport) {
-    stage = STAGE_SUMMARIZE;
+    stage = STAGE_SOAP_REVIEW;
   } else {
     stage = STAGE_EXPORT;
   }
 
+  const onComprehendResultDelete = (r) => {
```

Comment:
Let's flip the boolean for the ternary and remove console logs

Reply:
Done.